alert.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/228add9a8b.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="alert.css">
+    <title>Document</title>
```

Comment:
Change the title of the document to alert

Reply:
done! Thanks for the review